morphir/runtime/src/org/finos/morphir/runtime/quick/testing/UnitTesting.scala

```diff
@@ -205,4 +215,19 @@ object UnitTesting {
     tests
   }
 
+  /**
+   * The inverse of collect Tests - returns non test definitions
+   *
+   * @param globals
+   *   The global definitions to collect non tests from
+   */
+  private[runtime] def collectNonTests(
```

Comment:
This function should also exclude anything which references test code - i.e., if either a function or anything it references is from the Test package, we should not consider it for coverage methods (as it is presumably a test-only utility function)